vnext/Shared/DevSupportManager.cpp

```diff
@@ -232,6 +235,26 @@ void DevSupportManager::StopPollingLiveReload() {
   m_cancellation_token = true;
 }
 
+void DevSupportManager::startInspector(const std::string &packagerHost, const uint16_t packagerPort) {
+#if defined(HERMES_ENABLE_DEBUGGER)
+  auto packageStatics = winrt::get_activation_factory<winrt::Windows::ApplicationModel::IPackageStatics>(
+  winrt::name_of<winrt::Windows::ApplicationModel::Package>());
+  std::string packageName = winrt::to_string(packageStatics.Current().DisplayName());
+
+  auto hostNames = winrt::Windows::Networking::Connectivity::NetworkInformation::GetHostNames();
+  std::string deviceName = winrt::to_string(hostNames.First().Current().DisplayName());
```

Comment:
Can direct debugging happen across machines like we allow today for the bundler and app to be on diff machines?

Reply:
Yep.. that should work ..